### PR TITLE
Bugfixes

### PR DESCRIPTION
**Pull request by Moetez16**

### CODE REVIEW

File: server/gokbg3/grails-app/domain/org/gokb/cred/Package.groovy [hunk 1]
```diff
@@ -287,6 +287,16 @@ class Package extends KBComponent {
     result
   }
 
+  @Transient
+  public getTippCount() {
+    def combo_tipps = RefdataCategory.lookup(RCConstants.COMBO_TYPE, 'Package.Tipps')
+
+    int result = Combo.executeQuery("select count(c.id) from Combo as c where c.fromComponent = ? and c.type = ?"
```
Comment (croqueGrec09) [inline server/gokbg3/grails-app/domain/org/gokb/cred/Package.groovy:294]: using ? as argument is deprecated and against convention

File: server/gokbg3/grails-app/controllers/org/gokb/CreateController.groovy [hunk 2]
```diff
@@ -209,11 +209,16 @@ class CreateController {
                 }
               }
 
-             /* if ( flash.error.size() == 0 ) {
-                flash.error.add("There has been an error creating the component. Please try again.")
-              }*/
+              if(result.errors.size() > 0){
+                flash.error = result.errors
+                println("Moe")
```
Comment (croqueGrec09) [inline server/gokbg3/grails-app/controllers/org/gokb/CreateController.groovy]: Yay, the general developer eternalises himself!! :D

File: server/gokbg3/grails-app/controllers/org/gokb/AdminController.groovy [hunk 3]
```diff
@@ -457,14 +457,15 @@ class AdminController {
       Integer tippDuplicatesByNameCount = aPackage.getTippDuplicatesByNameCount()
       Integer tippDuplicatesByUrlCount = aPackage.getTippDuplicatesByURLCount()
       Integer tippDuplicatesByTitleIDCount = aPackage.getTippDuplicatesByTitleIDCount()
+      log.debug("Package ${aPackage.name} : ${index}")
```
Comment (croqueGrec09) [inline server/gokbg3/grails-app/controllers/org/gokb/AdminController.groovy:460]: log output needed?

File: server/gokbg3/grails-app/taglib/InplaceTagLib.groovy [hunk 4]
```diff
@@ -113,10 +113,10 @@ class InplaceTagLib {
       if (oid && (oid != '')) out << " data-pk=\"${oid}\""
       out << " data-name=\"${attrs.field}\""
 
-      // SO: fix for FF not honouring no-wrap css.
+/*      // SO: fix for FF not honouring no-wrap css.
```
Comment (croqueGrec09) [inline server/gokbg3/grails-app/taglib/InplaceTagLib.groovy:116]: leave in for?

**Review by croqueGrec09 — COMMENTED** [review]
remarks

File: server/gokbg3/grails-app/controllers/org/gokb/AdminController.groovy (hunk 3 above)
Comment (Moetez16) [reply server/gokbg3/grails-app/controllers/org/gokb/AdminController.groovy:460]: Yes. Please.

File: server/gokbg3/grails-app/taglib/InplaceTagLib.groovy (hunk 4 above)
Comment (Moetez16) [reply server/gokbg3/grails-app/taglib/InplaceTagLib.groovy:116]: History ;)

File: server/gokbg3/grails-app/domain/org/gokb/cred/Package.groovy (hunk 1 above)
Comment (Moetez16) [reply server/gokbg3/grails-app/domain/org/gokb/cred/Package.groovy:294]: Legacy

File: server/gokbg3/grails-app/controllers/org/gokb/CreateController.groovy (hunk 2 above)
Comment (Moetez16) [reply server/gokbg3/grails-app/controllers/org/gokb/CreateController.groovy]: Done

File: server/gokbg3/grails-app/domain/org/gokb/cred/Package.groovy (hunk 1 above)
Comment (croqueGrec09) [reply server/gokbg3/grails-app/domain/org/gokb/cred/Package.groovy:294]: done in separate process

**Review by croqueGrec09 — APPROVED** [review]
approved

**Pull request merged**